src/verification.rs

```diff
@@ -0,0 +1 @@
+//! A tool to be used by the client to perform verification on the final product.
```

Comment:
Let's call this file verifylib.rs? That'll keep it consistent with the runlib naming, as well as the other implementations.

Reply:
I think that makes sense! Thank you for the suggestion :D